src/utils/emailMessages.js

```diff
@@ -16,4 +16,10 @@ const verifyMessage = {
   text: 'Confirm'
 };
 
-export { resetMessage, changedMessage, verifyMessage };
+const userRoleMessage = {
+  intro: 'Cngratulations, you have been assigned to a role',
```

Comment:
1. I think here You wanted to write `Congratulations` and you misspelled.

2. Is this the message we send to a user when his/her role has been changed? If so, is this message complete? I think since you are notifying him/her about the role changes, you can precise the role which he/she got.

Reply:
> 1. I think here You wanted to write `Congratulations` and you misspelled.
> 2. Is this the message we send to a user when his/her role has been changed? If so, is this message complete? I think since you are notifying him/her about the role changes, you can precise the role which he/she got.

I no longer need that